modules/ding_permissions/ding_permissions.module

```diff
@@ -1011,3 +1013,14 @@ function ding_permissions_secure_permissions($role) {
     return $permissions[$role];
   }
 }
+
+/**
+ * Implements hook_page_alter().
```

Comment:
@cableman That's a decision by the business - you can [read more about it in the ticket tracker](http://platform.dandigbib.org/issues/2206).